containers/tefca-viewer/e2e/query_workflow.spec.ts

```diff
@@ -168,127 +179,18 @@ test.describe("querying with the TryTEFCA viewer", () => {
     page,
   }) => {
     await page.getByRole("button", { name: "Go to the demo" }).click();
-    await page.getByLabel("Query", { exact: true }).selectOption("chlamydia");
     await page.getByRole("button", { name: "Fill fields" }).click();
     await page.getByRole("button", { name: "Search for patient" }).click();
     await expect(page.getByText("Loading")).toHaveCount(0, { timeout: 10000 });
 
-    await expect(
-      page.getByRole("heading", { name: "Patient Record" }),
-    ).toBeVisible();
-  });
-});
-
-test.describe("Test the user journey of a 'tester'", () => {
-  test.beforeEach(async ({ page }) => {
-    // Start every test on direct tester page
-    await page.goto("http://localhost:3000/tefca-viewer/query/test", {
-      waitUntil: "load",
-    });
-  });
-
-  test("query/test page loads", async ({ page }) => {
```

Comment:
Deleted these since we removed the test page flow as discussed [here](https://skylight-hq.slack.com/archives/C05BEG226RZ/p1729013913939009)